### ISSUE TITLE
Abnormal circulating Monocyte Chemotactic Protein 1 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6001]
**Preferred term label:** Abnormal circulating Monocyte Chemotactic Protein 1 concentration

**Synonyms** Abnormal concentration of Monocyte chemoattractant protein-1 (MCP-1)

**Definition (free text, please give PubMed ID)** Concentration of monocyte chemotactic protein 1 in the blood is outside the normal range., PMID: 19441883

**Parent term (use hpo.jax.org/app)** Abnormality of serum cytokine level HP:0011112

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Monocyte Chemotactic Protein 1 concentration

**Synonyms** Increased concentration of Monocyte chemoattractant protein-1 (MCP-1)

**Definition (free text, please give PubMed ID)** Concentration of monocyte chemotactic protein 1 in the blood is below the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Monocyte Chemotactic Protein 1 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Monocyte Chemotactic Protein 1 concentration

**Synonyms** Increased concentration of Monocyte chemoattractant protein-1 (MCP-1)

**Definition (free text, please give PubMed ID)** Concentration of monocyte chemotactic protein 1 in the blood is above the normal range.

**Parent term (use hpo.jax.org/app)** Abnormal circulating Monocyte Chemotactic Protein 1 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
No normal ranges have been determined for this protein in serum. For instance, there is no LOINC term for this test. Therefore, it is possible to do a cohort study that shows a higher level in individuals with some condition, but it is not possible to do a test in an individual and say that the level is "too high" or "too low". This means that this would not be a valid HPO term.
